x/staking/keeper/keeper.go

```diff
@@ -61,8 +61,8 @@ func NewKeeper(
 		cdc:                cdc,
 		storeKey:           storeKey,
 		paramSubspace:      paramstore,
-		chainKeeper:        chainKeeper,
-		moduleCommunicator: moduleCommunicator,
+		ChainKeeper:        chainKeeper,
+		ModuleCommunicator: moduleCommunicator,
```

Comment:
I am not really comfortable with exporting `ModuleCommunicator`. @jdkanani ?

Reply:
That's tricky since side_handler is not in keeper package. Either we can keep it in keeper to avoid exporting moduleCommunicator, similar to normal handlers?

Reply:
normal handlers are also in different package (in <module> package). But let's not block on this, I will create a tracking issue and will see if we can find a better solution.

Reply:
https://github.com/maticnetwork/heimdall/issues/469